wd_core.au3

```diff
@@ -174,6 +174,7 @@ Global $_WD_RESPONSE_TRIM = 100 ; Trim response string to given value for debug
 Global $_WD_ERROR_MSGBOX = True ; Shows in compiled scripts error messages in msgboxes
 Global $_WD_DEBUG = $_WD_DEBUG_Info ; Trace to console and show web driver app
 Global $_WD_CONSOLE = ConsoleWrite ; Destination for console output
+Global $_WD_CONSOLE_Sufix = @CRLF ; Suffix added to the end of Message in $_WD_CONSOLE function

```

Comment:
There should be two `f`s in suffix

Reply:
Take a look on: [small fixes](https://github.com/Danp2/au3WebDriver/pull/242/commits/070aec7f1e04fa8069fdb99340a198018125a5ac)